gtfs.js

```diff
@@ -1620,6 +1620,74 @@ class Gtfs {
    */
   setIndexedTransfers(indexedTransfers) { setTable(this, 'transfers', indexedTransfers); }
 
+  /* pathways.txt */
+
+  /**
+   * Adds a pathway in the GTFS.
+   *
+   * @param {Object} pathway Pathway to add in the GTFS.
+   */
+  addPathway(pathway) { addItems(this, 'pathways', [pathway]); }
+
+  /**
+   * Adds a list of pathways in the GTFS.
+   *
+   * @param {Array.<Object>} pathways Array of transfers to add in the GTFS.
+   */
+  addPathways(pathways) { addItems(this, 'pathways', pathways); }
+
+  /**
+   * Apply a function to each pathway in the GTFS.
+   *
+   * @param {function} iterator Function which will be applied on every pathway.
+   */
+  forEachPathway(iterator) { forEachItem(this, 'pathways', iterator); }
+
+  /**
+   * Get the indexed pathways of the GTFS. The indexation is defined in the schema (see schema.js).
+   *
+   * @return {Map.<string, Map.<string, Object>>} Indexed transfers.
+   */
+  getIndexedPathways() { return getIndexedTable(this, 'pathways'); }
+
+  /**
+   * Get a pathway using its indexes: the fromStopId and the toStopId.
```

Comment:
You say using two indexes but then only use the pathwayId. You should update this jsDoc